### DISCUSSION TITLE
Serial not working

### DISCUSSION THREAD

**Discussion by vedantk91** [dbuezas/lgt8fx#216] [General]
I am using the chip. Connecting it with a UNO for programming. Trying to do serial.print by connecting TX,RX but serial is not working.

**Comment by LaZsolt** [comment]
> Connecting it with a UNO for programming.

What do you mean? Did you want to program LGT8F328P with an UNO? In this case I recommend you this document: https://github.com/SuperUserNameMan/LGTISP
